docs/reference/modules/snapshots.asciidoc

```diff
@@ -24,7 +24,7 @@ Elasticsearch. The repository settings are repository-type specific. See below f
 
 [source,js]
 -----------------------------------
-PUT /_snapshot/my_backup
+PUT _snapshot/my_backup
 {
   "type": "fs",
   "settings": {
```

Comment:
You could do `// CONSOLE` and `// TESTESTUP` and that way the repository is created for every snippet.